examples/invoke.rs

```diff
@@ -0,0 +1,62 @@
+extern crate parity_wasm;
+
+use std::env::args;
+
+use parity_wasm::{interpreter, ModuleInstanceInterface, RuntimeValue};
+use parity_wasm::elements::ExportEntry;
+use parity_wasm::elements::ExportSection;
+use parity_wasm::elements::FunctionSection;
+use parity_wasm::elements::TypeSection;
+use parity_wasm::elements::Internal;
+use parity_wasm::elements::Type;
+use parity_wasm::elements::FunctionType;
+use parity_wasm::elements::ValueType;
+
+fn main() {
+    let args: Vec<_> = args().collect();
+    if args.len() < 3 {
+        println!("Usage: {} <wasm file> <exported func> [<arg>...]", args[0]);
+        return;
+    }
+    let func_name = &args[2];
+    let (_, program_args) = args.split_at(3);
+
+    let program = parity_wasm::ProgramInstance::with_env_params(
+        interpreter::EnvParams {
+            total_stack: 128*1024,
+            total_memory: 2*1024*1024,
+            allow_memory_growth: false,
+        }
+    ).expect("Failed to load program");
+
+    let module_def = parity_wasm::deserialize_file(&args[1]).expect("Failed to load module");
+    let execution_params = {
+        let export_section: &ExportSection = module_def.export_section().expect("No export section found");
```

Comment:
`:&ExportSection` explicit declaration is not needed

here and a lot of the same below